work_queue/src/bindings/python3/PyTask/py_wrapper.py

```diff
@@ -0,0 +1,24 @@
+if __name__=='__main__':
```

Comment:
Ok, this is the right file, but you don't want it to live in the source tree.  What the PyTask module should do is create a file with these contents in the current working directory, at runtime.